Assignment2/VictorNS69-Victor Nieves.ttl

```diff
@@ -0,0 +1,8 @@
+:Class01 includes :Sensor029;
+	includes :Computer101.
+:Computer101 hasOwner :User10A.
+:User10A hasName :"Pedro".
+
+:Sensor029 hasMeasurement :Measurement8401.
+:Measurement8401 hasTemperature :"29";
```

Comment:
Literals should be datatyped, and in fact the syntax here is wrong.

Reply:
Changed in the last commit